doc/guide/guide-basics.rst

```diff
@@ -123,7 +125,7 @@ Notice how both the dims and shape change according to the input data.  Although
 States and operators
 ---------------------
 
-Manually specifying the data for each quantum object is inefficient. Even more so when most objects correspond to commonly used types such as the ladder operators of a harmonic oscillator, the Pauli spin operators for a two-level system, or state vectors such as Fock states. Therefore, QuTiP includes predefined objects for a variety of states:
+Manually specifying the data for each quantum object is inefficient, since most objects correspond to commonly used types such as the ladder operators of a harmonic oscillator, the Pauli spin operators for a two-level system, or state vectors such as Fock states. Therefore, QuTiP includes predefined objects for a variety of states and operators:
```

Comment:
I think it was a little bit more clear before. Currently it suggests that states and operators are in the same table but they are not.